client/signup/jetpack-connect/index.jsx

```diff
@@ -68,14 +81,29 @@ const JetpackConnectMain = React.createClass( {
 	},
 
 	onURLEnter() {
+		this.props.recordTracksEvent( 'calypso_jpc_url_submit', {
+			jetpack_funnel: this.state.currentUrl,
```

Comment:
Why is this called `jetpack_funnel`? Seems like it should just be `jetpack_url` or `url` or something?